JAXRS/src/main/java/Recipes/Ingredient.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package Recipes;
+
+/**
+ *
+ * @author Single-eye
+ */
+public class Ingredient {
+    
+private String ingredientName; 
+    
+ public Ingredient(String inIngredient){
+     

```

Comment:
also needs to have an ID, like the recipe. Otherwise, we could just remove this class and use an arraylist of strings for the recipe instead.